### ISSUE TITLE
More easily navigate between Cluster Explorer, cluster detail and cluster config

### ISSUE THREAD

**Issue by catherineluse** [rancher/dashboard#4306]
I had some feedback based on my experience of stepping through every 2.6 feature while rewriting every set of instructions in the docs for Rancher 2.6. Here are some things I thought would have helped make it easier to navigate.

One thing that threw me off when I first tried documenting how to configure clusters was that I first tried to go to the cluster explorer page because I expected that to be a source of truth for everything related to that cluster. I also did the opposite, clicking the name of the cluster in the cluster list view when I wanted cluster explorer, then finding myself in the detail page, then having to go back. I found it counterintuitive that to edit the configuration of the cluster, you have to go to the global view first instead of being able to stay within a cluster section. I don't think you should have to go up in the hierarchy to make a lateral move.

- In Cluster Explorer, there should be a link to Edit Config and a link to the cluster actions so that anything you can do from the list view of cluster manager (rotate certificates etc) you can also reach from Cluster Explorer.
- From the cluster detail page there should be a link to Cluster Explorer.
- Fix the disappearing/reappearing tabs issue (see below).

**Comment by catherineluse** [comment]
I also don't like that the tabs on the cluster detail page rerender (visibly disappearing and reappearing) when you navigate between them. The tabs should stay there without rerendering while the content under them changes
<img width="1208" alt="Screen Shot 2022-02-07 at 3 31 59 PM" src="https://user-images.githubusercontent.com/20599230/152883353-05481cbc-366f-4044-a671-c3259e49f1f9.png">
.

**Comment by rak-phillip** [comment]
> I also don't like that the tabs on the cluster detail page rerender (visibly disappearing and reappearing) when you navigate between them. The tabs should stay there without rerendering while the content under them changes <img alt="Screen Shot 2022-02-07 at 3 31 59 PM" width="1208" src="https://user-images.githubusercontent.com/20599230/152883353-05481cbc-366f-4044-a671-c3259e49f1f9.png">

I just want to add that I came across this the other day and it's a very confusing experience

**Comment by catherineluse** [comment]
Yeah, it's confusing, especially when you navigate from YAML to Config. In that case the tabs linger for a moment, then disappear entirely.

**Comment by kwwii** [comment]
We should add the left action menu from the cluster management page to each cluster in the dashboard page.
